mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -524,6 +524,23 @@ VectorizationState::maskOperation(RewriterBase &rewriter, Operation *opToMask,
 
   if (!mask) {
     LDBG() << "No mask required";
+    if (assumeDynamicDimsMatchVecSizes) {
+      LDBG() << "Assuming dynamic dimensions match vector sizes!";
```

Comment:
I would move this inside the lambda. Otherwise it will be printed for every Op, which could be noise, no?
```suggestion
      LDBG() << "Assuming dynamic dimensions match vector sizes, set in_bounds to true!";
```